src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -573,6 +579,32 @@ public Void caseProxyObjectType(ProxyObjectType type) {
       });
     }
 
+    // This method is used for objects like the static props on an interface, that
+    // behave like an object with bunch of props, but are not technically defined as
+    // record types.
+    private void visitRecordLikeType(ObjectType type) {
+      emit("{");
+      Iterator<String> it = type.getOwnPropertyNames().iterator();
+      Boolean isFirst = true;
+      while (it.hasNext()) {
+        String propName = it.next();
+
+        // The prototype key is implicitly defined in the Object signature.
+        if (propName.equals("prototype")) continue;
+
+        if (isFirst) {
```

Comment:
leaves a comma if the last one is going to be skipped because it is 'prototype'. Iterators can't peek the next value as far as I can tell, so I change the logic to add the comma before each output, instead of after.